packages/compiler-cli/test/compliance/test_cases/r3_compiler_compliance/elements/security_sensitive_constant_attributes.js

```diff
@@ -0,0 +1,11 @@
+consts: [["src", i0.ɵɵtrustConstantResourceUrl `https://angular.io/`], ["srcdoc", i0.ɵɵtrustConstantHtml `<h1>Angular</h1>`], ["data", i0.ɵɵtrustConstantResourceUrl `https://angular.io/`, "codebase", i0.ɵɵtrustConstantResourceUrl `/`], ["src", "https://angular.io/"]],
```

Comment:
nit: we can use newlines here to make it a bit more readable:

```suggestion
consts: [
  ["src", i0.ɵɵtrustConstantResourceUrl `https://angular.io/`],
  ["srcdoc", i0.ɵɵtrustConstantHtml `<h1>Angular</h1>`],
  ["data", i0.ɵɵtrustConstantResourceUrl `https://angular.io/`, "codebase", i0.ɵɵtrustConstantResourceUrl `/`], 
  ["src", "https://angular.io/"]
],
```

Reply:
Fixed, thanks for all the great feedback @AndrewKushnir! :)

Reply:
Wlcm :) Always glad to help!